### ISSUE TITLE
SendTrap truncates last PDU

### ISSUE THREAD

**Issue by sbowman** [soniah/gosnmp#86]
The logic you're using to prepend the list of PDUs with a time tick is incorrect.  You're basically snipping the last PDU off the end of the list sent to SendTrap:

```
// add a timetick to start, set to now
now := uint32(time.Now().Unix())
timetickPDU := SnmpPDU{"", TimeTicks, now, x.Logger}
// prepend timetickPDU
copy(pdus[1:], pdus) // *** THIS DOES NOT EXTEND pdus!!! ***
pdus[0] = timetickPDU
```

What you probably meant:

```
// first lengthen pdus to accommodate the timetick
pdus = append(pdus, SnmpPDU{})
copy(pdus[1:], pdus)
pdus[0] = timetickPDU
```

**Comment by soniah** [comment]
@sbowman thanks for that. I'll look at this and your other issue.

**Comment by soniah** [comment]
Closed by Joshua Green with commit 413c1e91638fce965a27ac0d5c2c7c32f2d21036.
